iron-scroll-threshold.html

```diff
@@ -80,6 +80,7 @@
   <template>
     <style>
       :host {
+        -webkit-overflow-scrolling: touch;
```

Comment:
Should we set `-webkit-overflow-scrolling: touch` only when the host is the `scrollTarget`?  `-webkit-overflow-scrolling: touch` creates a stacking context and we might want to avoid this if the host is not the scrollable region.